recipe/build_opencv.sh

```diff
@@ -15,12 +15,11 @@ if [[ "${target_platform}" == linux-* ]]; then
 fi
 
 if [[ "$build_variant" == "normal" ]]; then
-    echo "Building normal variant"
+    echo "Building normal variant with Qt6"
     QT="6"
```

Comment:
```suggestion
    echo "Building normal variant with Qt${QT}"
```
And then you can set this variable in the recipe to the major version of Qt we currently support so when 7 comes out we don't have to hunt down all the places we've used 6 to update. Same if we want to produce a new build using 5 for some reason.
```
outputs:
  - name: {{ package_name }}
    build:
      script_env:
        - QT={{ qt.split(".")[0] }}  # [build_variant == "normal"]
        - QT=0                       # [build_variant == "headless"]
```

Reply:
Done